packages/hello-gsm/src/Stores/StoreContainer.ts

```diff
@@ -24,13 +27,16 @@ const useStore = create<storeType>(set => ({
   mypageModalContent: '',
   isSearching: false,
 
+  showSideBar: null,
```

Comment:
```suggestion
  isSearching: false,
  showSideBar: null,
```

이 부분 또한 관심사에 맞게끔 배치를 해보는게 어떨까요??